mantle/cmd/ore/aws/upload.go

```diff
@@ -80,6 +81,7 @@ func init() {
 	cmdUpload.Flags().StringVar(&uploadAMIName, "ami-name", "", "name of the AMI to create")
 	cmdUpload.Flags().StringVar(&uploadAMIDescription, "ami-description", "", "description of the AMI to create (default: empty)")
 	cmdUpload.Flags().StringSliceVar(&uploadGrantUsers, "grant-user", []string{}, "grant launch permission to this AWS user ID")
+	cmdUpload.Flags().StringSliceVar(&uploadSnapshotUsers, "snapshot-user", []string{}, "grant snapshot volume permission to this AWS user ID")
```

Comment:
WDYT about `--grant-user-snapshot` or something similar? (And maybe eventually we should rename `--grant-user` to `--grant-user-launch` or something).

Reply:
sgtm. Didn't want to change up too much here, but wasn't too happy with the `grant-user` naming either :)

Reply:
I've changed this to `grant-user-snapshot` now (`grant-user-volume` may also make sense). Renaming the existing `grant-user` to `grant-user-launch` is not included here. Should it be?